docs/source/checks/tabular/methodology/plot_boosting_overfit.py

```diff
@@ -2,8 +2,40 @@
 """
 Boosting Overfit
 ****************
-Load Data
-=========
+
+This notebooks provides an overview for using and understanding feature drift check.
+
+**Structure:**
+
+* `What is a boosting overfit? <#what-is-a-boosting-overfit>`__
+* `Generate data & model <#generate-data-model>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+What is A Boosting Overfit?
+===========================
+A boosting algorithm is a machine learning algorithm that uses a combination of weak learners to predict a target
+variable. The mechanism of boosting is to increase the number of weak learners in the ensemble by adding a new
+weak learner to the ensemble. The new weak learner is trained on the data that is not predicted correctly by the
+ensemble in the previous iteration. The mechanism continues until the ensemble reach a certain level of accuracy or
+until the number of iterations is reached.
+
+Thanks to its mechanism, boosting algorithms are usually less prone to overfitting. However, the number of weak
+learners in the ensemble can be too large making the ensemble too complex. In this case, the ensemble may be
+overfitted on the training data.
+
+How deepchecks detects a boosting overfit?
+------------------------------------------
+The check runs for a pre-defined number of iterations, and in each step it limits the boosting model to use up to X
```

Comment:
Not clear enough that we use the estimators in the existing model rather than training a new one.